README.md

```diff
@@ -20,7 +20,7 @@ It can be found [here](https://github.com/SubstraFoundation/substra)
 
 You will need to install it thanks to the `pip` binary.
 
-# Run the tests
+# Run the tests on the default backend
```

Comment:
I'd say "on a remote backend" since the section provides explanation for both default and alternate remote backends.